src/Home.js

```diff
@@ -20,30 +27,33 @@ function Home() {
                     <FaCog className="settings-button" />
                 </Link>
                 <small></small><br />
-                <Link to="/leaderboards">
+                <Link to="/leaderboard">
                     <FaListOl className="leaderboard-button" />
                 </Link>
-                <div className="playContainer">
-                    <Link to="/skins">
-                        <FaPalette className="skins-button" />
-                    </Link>
+                <div className="movingGradient">
```

Comment:
The moving gradient is very impressive and took our UI to the next level.

Reply:
Appreciate it! Just wanted to play around with CSS and see what I could do.